app/models/user.rb

```diff
@@ -104,6 +104,10 @@ def self.find_user_by_mobile_or_email(mobile, email)
     end
   end
 
+  def app_name(user, app_name)
+    user.request_from_stock = (app_name == "stock")
```

Comment:
im not sure of this needs to be done this way, as the operation is dependent on the type of app, rather than roles/permission

cc: @abulsayyad123 @patrixr @swatijadhav

Reply:
Yup... previously we had talk on this that we have to remove dependency on `app` as much as we can.